### PR TITLE
SDK-288 [Java SDK] add Schema 2.3 support

### PR DESCRIPTION
**Pull request by VladimirLevitskiy**
Pull Request Checklist
======================

Please read this for our PR Culture:
------------------------------------
https://incountry.atlassian.net/wiki/spaces/ED/pages/46858869/Pull+requests+culture+Draft

Work relates to:
----------------
Link(s) to Jira Task/User Story/Bug/Epic:
- [SDK-288](https://incountry.atlassian.net/browse/SDK-288)

Code Quality (Before submitting PR)
------------
- [x] Ensure code compiles and passes Sonar Lint.
- [x] Do your best to ensure integration tests pass. If they don't then...
- [x] Remove, ignore, or mark pending any brittle tests.
- [x] Remove commented out code.

Target Release
--------------
- [ ] Please put the target release vehicle this needs to be bundled with.
- [ ] Did you increment the version? What is the new version?

### CODE REVIEW

**Review by robert-par — DISMISSED** [review]

File: src/main/java/com/incountry/residence/sdk/tools/JsonUtils.java [hunk 1]
```diff
@@ -75,10 +87,12 @@
 
     private static final List<String> REMOVE_KEYS = Arrays.asList(P_BODY, P_PRECOMMIT_BODY, P_CREATED_AT, P_UPDATED_AT,
```
Comment (RommeroInCountry) [inline src/main/java/com/incountry/residence/sdk/tools/JsonUtils.java:88]: Let's think of a solution to get rid of this approach in future. It's kinda strange-looking :)

**Review by RommeroInCountry — CHANGES_REQUESTED** [review]
Please mention new keys in Readme in **List of available record fields** section

File: src/main/java/com/incountry/residence/sdk/tools/JsonUtils.java (hunk 1 above)
Comment (VladimirLevitskiy) [reply src/main/java/com/incountry/residence/sdk/tools/JsonUtils.java:88]: It will be done during task https://incountry.atlassian.net/browse/SDK-291

**Review by RommeroInCountry — APPROVED** [review]
LGTM

**Review by robert-par — APPROVED** [review]

**Pull request merged**